collections/_tutorials/tutorial-optimal-transport.md

```diff
@@ -0,0 +1,358 @@
+$$
+
+where $$T$$ is a transport map from $$X$$ to $$Y$$ and $$\Gamma(\mu, \nu)$$ denotes the collection of all probability measures on $$X \times Y$$ with marginals $$\mu$$ on $$X$$ and $$\nu$$ on $$Y$$ (continuous equivalent of $$U(\mu, \nu)$$ above). 
+
+<div style="text-align:center">
+<img src="/collections/images/optimal_transport/discrete_semi_continuous.jpg" width=600></div>
+<p style="text-align: center;font-style:italic">Figure 3. Examples of transort plans in the discrete, semidiscrete and continuous cases.</p>
+
+&nbsp;
+
+### Wasserstein distance and barycenter
+
+For $$p \in [1, +\infty]$$ and $$d(\cdot, \cdot)$$ a distance, the following quantity define a distance between two distributions $$\mu$$ and $$\nu$$ :
+
+$$
+W_p(\mu, \nu) = (\underset{\gamma \in \Gamma(\mu, \nu)}{\inf} \int_X d^p(x,y) \mathrm{d} \gamma(x,y))^{\frac{1}{p}}
+$$
+
+Given a set of measure $$\{b_s\}_{s=1,...,S}$$, one way to define the barycenter of these measures is :
+
+$$
+\underset{a}{\min} \sum_{s=1}^{S} \lambda_s W_p^p(a,b_s)
+$$
+
+where $$\lambda_1,...,\lambda_S$$ are the weights given to each distribution.
+
+&nbsp;
+
+### **Limits for practical applications**
+
+Although properties of existence and uniqueness have been demonstrated for optimal transport for the formulation of Kantorovich, as well as equivalence between Monge and Kantorovich's formulation under certain conditions, there remains several major practical limitions for optimal transport to be directly applied in large dimension :
+- If the source and target distributions respectively have respectively $$m$$ and $$n$$ supports points, the solution of optimal transport can be found at best at cost $$O((n+m)nm\log(n+m))$$, which is way to expensive for large datasets.
+- The optimal transport plan can be "noisy" or "irregular" with respect to inputs.
+- The optimal solution $$P^*$$ may not be unique and has no meaningful Jacobian with respect to inputs $$X$$ or $$Y$$, as illustrated if Figure 4.
+- The optimal transport plan is bound to the points that are given when its computed. One important question is what happens when a new point is given.
+-  One may want to compute an optimal transport between distributions that don't live in the same space or that different mass (non equal to 1). It that possible and how to do it ?
+-  The computation of Wasserstein distance suffers from the curse of dimensionality.
+
+<div style="text-align:center">
+<img src="/collections/images/optimal_transport/non_differentiable.jpg" width=400></div>
+<p style="text-align: center;font-style:italic">Figure 4. Illustration of a limitation of optimal transport for modern applications.</p>
+
+
+All these questions are crucial for optimal transport to be applied for large dataset and in deep learning. It is because they have found satisfying answers that optimal transport is now applied for few years in these domains. The next section aims to present some solutions to the problems listed above.
```

Comment:
```suggestion
All these questions are crucial for optimal transport to be applied to large datasets and deep learning. It is because satisfying answers have been found that optimal transport has now been in use for a few years in these domains. The next section aims to present some solutions to the problems listed above.
```